### DISCUSSION TITLE
Is there a way to visually stress newly added/changed nodes include full branch of parents?

### DISCUSSION THREAD

**Discussion by kubroid** [vakata/jstree#2684] [Q&A]
Like temporary change style, blink, or ...

PS: thanks for such a great library!
PPS: sorry for the possibly dumb question, I'm a newbie in HTML/JS.

**Comment by vakata** [comment]
Sorry, currently there is no way to do this out of the box. There are events you could use - like create_node.jstree and rename_node.jstree, if you listen for those you could temporarily set a class on the node and have it do whatever you want using CSS.
